src/components/Albums/Albums.js

```diff
@@ -84,13 +24,25 @@ export class AlbumsComponent {
     });
   }
 
-  mount() {
+  fetchAlbumsCollectionData() {
+    this.state.isFetching = true;
```

Comment:
do you need this `isFetching` ?? we do not show progress bar currently - please remove it

Reply:
Done